auth/src/lib.rs

```diff
@@ -35,6 +35,27 @@ pub mod login {
         pub server_data: String,
         pub credential_finalization: opaque::client::login::CredentialFinalization,
     }
+
+    #[derive(Serialize, Deserialize, Clone)]
+    pub struct ClientConventionalLoginRequest {
+        pub username: String,
+        pub password: String,
+    }
+
+    #[derive(Serialize, Deserialize, Clone)]
+    pub struct ServerLoginResponse {
+        pub token: String,
+        #[serde(skip_serializing_if = "String::is_empty")]
+        #[serde(rename = "refreshToken")]
+        pub refresh_token: String,
+    }
+
+    #[derive(Serialize, Deserialize, Clone)]
+    pub struct ServerResetResponse {
```

Comment:
Server reset? What's that for? Maybe we need a better name.

Reply:
Renamed it to password_reset::ServerPasswordResetResponse